force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,36 +219,45 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
+        
+        String result = '';
 
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
+        
+        if (repeatCount <= 0 || inputString == null) {
+            return ''; 
+        }
 
-            // If it is not the last iteration, add a semicolon to separate the strings
+       
+        for (Integer i = 0; i < repeatCount; i++) {
+            result += inputString; 
+            
+            if (i < repeatCount - 1) {
+                result += '; '; 
+            }
         }
-        */
 
-        // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return result; 
     }
 
+        
+    
+    
+
     /**
      * Question 11
      * Creates a List of integers and adds the integers 1, 2, and 3 to it.
      * Example: createAndPopulateList() should return [1, 2, 3]
      * @return A List of integers containing the numbers 1, 2, and 3.
      */
     public static List<Integer> createAndPopulateList() {
-        // Create a new list of integers
+        List<Integer> integerList = new List<Integer>();
 
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
+        integerList.add (1);
+        integerList.add (2);
+        integerList.add (3);
```

Comment:
try to instantiate the list in the constructor
https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex